src/dmd/backend/dwarfdbginf.d

```diff
@@ -2264,82 +2261,62 @@ static if (1)
         type *tbase;
         const(char)* p;
 
-        static immutable ubyte[10] abbrevTypeBasic =
-        [
-            DW_TAG_base_type,       DW_CHILDREN_no,
-            DW_AT_name,             DW_FORM_string,
-            DW_AT_byte_size,        DW_FORM_data1,
-            DW_AT_encoding,         DW_FORM_data1,
-            0,                      0,
-        ];
-        static immutable ubyte[10] abbrevWchar =
```

Comment:
was this unused?

Reply:
Not unused, but duplicated with the above. I'm reusing the same abbreviation for basic types: `abbrevTypeBasic`.

Reply:
Ah, OK